extensions/CheckUser/api/ApiQueryCheckUserLog.php

```diff
@@ -40,10 +46,10 @@ public function execute() {
 		foreach ( $res as $row ) {
 			$log[] = array(
 				'timestamp' => wfTimestamp( TS_ISO_8601, $row->cul_timestamp ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "timestamp" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)